tests/cql/CqlDateTimeOperatorsTest.xml

```diff
@@ -563,15 +568,15 @@
 	<group name="Uncertainty tests">
 		<test name="DateTimeDurationBetweenUncertainInterval">
 			<expression>days between DateTime(2014, 1, 15) and DateTime(2014, 2)</expression>
-			<output>Interval[ 17, 44 ]</output>
+			<output>Interval[ 16, 44 ]</output>
```

Comment:
* lower: (whole) days between @2014-01-15T23:59:59.999 and @2014-02-01T00:00:00.000 = `16`
* upper: (whole) days between @2014-01-15T00:00:00.000 and @2014-02-28T23:59:59.999 = `44`